src/controllers/movies/movies.controller.ts

```diff
@@ -0,0 +1,101 @@
+import { type Request, type Response } from 'express'
+import Movie from '../../models/movie';
+import mongoose from 'mongoose';
+const ObjectId = mongoose.Types.ObjectId;
+
+//Movie creation controller
+const postMoviesResponse = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const newMovie = req.body;
+        if (Object.keys(newMovie).length === 0) {
+            res.status(400).json({ error: 'Request body is missing' });
+            return;
+        }
```

Comment:
Better to add some validations for checking not just body length. For example, current implementation allow to add movie with genres that doesn't exists in genres table. You can or validate such presence and throw error that current genres not available and/or implement functionality for adding new (not yet existing) genres. The first implementation flow is more convenient.